ex03.ex

```diff
@@ -55,9 +55,17 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
-
+  def odd_even([h | t]) when Integer.is_even(h) do
```

Comment:
## -1

You could take this a step further and remove a little duplication with

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~